autograder/submitty_autograding_shipper.py

```diff
@@ -37,6 +37,58 @@
 JOB_ID = '~SHIP~'
 
 
+# This function is copied, would rather use the version in this file instead.
+# import sys
+# sys.path.append('/usr/local/submitty/sbin/shipper_utils/')
+# import ssh_jump_proxy
```

Comment:
If you do go this route, you're probably gonna want to replace
```python
import ssh_jump_proxy
```
With:
```python
from shipper_utils import ssh_jump_proxy
```

Reply:
changed!